include/wx/gtk/textentry.h

```diff
@@ -100,6 +101,7 @@ class WXDLLIMPEXP_CORE wxTextEntry : public wxTextEntryBase
     // GtkEntry IM context.
     int GTKEntryIMFilterKeypress(GdkEventKey* event) const;
 
+    int m_maxlen = 0;
```

Comment:
Is there a reason this is `protected` instead of `private`?
If so, maybe separate it from the unrelated comments below. If not, move it to the other variable declarations.

Reply:
@MaartenBent ,
I don't remember.

I think it should be private.

Will fix.

Reply:
@MaartenBent ,
It is declared protected because SetMaxLength is a function inside wxTextCtrl to set it.

For now I will keep it protected. I might change it when I go to Mac implementation and put it in the main header...

Reply:
I think generally you make something private unless there is a reason that it should be protected or public.